Makefile

```diff
@@ -70,7 +70,7 @@ all: static-analysis test ## Run all linux-based checks (linting, license check,
 static-analysis: check-go-mod-tidy lint check-licenses check-json-schema-drift  ## Run all static analysis checks
 
 .PHONY: test
-test: unit integration validate-cyclonedx-schema benchmark cli ## Run all tests (currently unit, integration, linux compare, and cli tests)
+test: unit integration validate-cyclonedx-schema benchmark test-utils cli ## Run all tests (currently unit, integration, linux compare, and cli tests)
```

Comment:
I believe this causes Python to be needed for `make test`, when it wasn't previously. Should that be added to https://github.com/anchore/syft/blob/main/DEVELOPING.md#getting-started ? (Maybe as an optional requirement since running the whole `make test` is relatively unusual.)

Reply:
there technically was an implicit requirement on python for coverage analysis, but that just means it should have already been in the doc 😄  (I'll add it)